src/index.js

```diff
@@ -6,13 +6,16 @@ import parser from './parser'
 import doApiCall from './api';
 
 let logger = log4js.getLogger();
-logger.level = 'debug';
+logger.level = process.env.DEBUG_LEVEL || 'warn';
 
 const Influx = require('influx');
 // InfluxDB Initialization.
 const influx = new Influx.InfluxDB({
 	host: process.env.INFLUX_URL,
-	database: process.env.INFLUX_DB
+	database: process.env.INFLUX_DB,
+	username: process.env.INFLUX_USER,
```

Comment:
Credentials should be optional, I don't personally use them as my database is in a private network.
Can you provide a solution where no user and password is passed ?

Reply:
I agree, but I am not sure what would be the typical way of achieving this. Should I do " || (empty string) "?

Reply:
According to this https://node-influx.github.io/typedef/index.html#static-typedef-ISingleHostConfig the values both defaults to 'root'. So maybe it still works even if there is no variable set ?

Reply:
I just tested it with your code and it seems to work even if we don't provide a username or a password.

Reply:
i believe this is sent as a basic auth header, the server probably just ignores it if there is no authentication configured